### PR TITLE
[lr] Use new method to generate LabeledPoint data

### PR DESCRIPTION
**Pull request by ynXiang**
Since, originally, LabeledPoint data generator is generated by decision tree from spark-perf. It is a little complicated. As a result, use LabeledPoint data generator from Spark, which is simple.

### CODE REVIEW

**Comment by carsonwang** [comment]
@VinceShieh @mpjlu can you please review this ?

File: sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/LogisticRegression.scala [hunk 1]
```diff
@@ -28,16 +28,14 @@ object LogisticRegression {
...
-    if (args.length == 2) {
+    if (args.length == 1) {
       inputPath = args(0)
-      maxresultsize = args(1)
     }
 
     val conf = new SparkConf()
 	.setAppName("LogisticRegressionWithLBFGS")
-	.set("spark.driver.maxResultSize", maxresultsize)
+	//.set("spark.driver.maxResultSize", maxresultsize)
```
Comment (VinceShieh) [inline sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/LogisticRegression.scala]: remove this comment?

File: conf/hibench.conf [hunk 2]
```diff
@@ -3,10 +3,10 @@
 hibench.scale.profile                 tiny
 
 # Mapper number in hadoop, partition number in Spark
-hibench.default.map.parallelism         8
+hibench.default.map.parallelism         138
```
Comment (VinceShieh) [inline conf/hibench.conf]: change this value to default and doc

File: conf/hibench.conf [hunk 3]
```diff
@@ -3,10 +3,10 @@
...
 
 # Reducer nubmer in hadoop, shuffle partition number in Spark
-hibench.default.shuffle.parallelism     8
+hibench.default.shuffle.parallelism     138
```
Comment (VinceShieh) [inline conf/hibench.conf]: same as above

**Comment by ynXiang** [comment]
problems fixed @VinceShieh

**Comment by VinceShieh** [comment]
thanks. 
LGTM

**Comment by carsonwang** [comment]
Merged. Thanks.

**Pull request merged**